app/src/main/res/layout/activity_main.xml

```diff
@@ -7,12 +7,26 @@
     tools:context=".MainActivity">
 
     <TextView
+        android:id="@+id/parking_textview"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:text="Hello World!"
+        android:text="@string/parking_text"
         app:layout_constraintBottom_toBottomOf="parent"
         app:layout_constraintLeft_toLeftOf="parent"
```

Comment:
instead of using 
        app:layout_constraintLeft_toLeftOf="parent"
you should use
        app:layout_constraintStart_toStartOf="parent"

Reply:
Done.